### ISSUE TITLE
Pre competition storage trip

### ISSUE THREAD

**Issue by RealOrangeOne** [srobo/tasks#995]
We'll need to visit storage at some point before the competition, to check quantities of consumables (tape etc.) and to pick up other things which need checking/fixing/updating etc. This task exists to track the things we should do on that trip.

 - [x] Collect the display & compbox Pis for checking & flashing, should be in RUB sr0J62 (grab the whole box)
 - [x] Check for robot flag fittings
 - [x] Check for robot flags
 - [x] Check tape quantities
 - [x] Check on stationery (pens and clipboards)
 - [x] Check on high-vis jackets (#1024)
 - [x] Check for arena markers (#1016)
 - [x] Check for table cloths (#1003)
 - [x] Red Lanyards for Media Consent

Please do add to the above list.

### Original

[comp/pre-comp-storage-trip](https://github.com/srobo/recurring-tasks/blob/master/comp/pre-comp-storage-trip.yaml)

**Comment by Tyler-Ward** [comment]
I think there are some high vis jackets in my garage as they were in team kits, need to confirm however

**Comment by RealOrangeOne** [comment]
I will likely be doing a container trip either mid Feb or March to stock check (**not** to grab things in preparation). If it's on this issue, I'll check it.

**Comment by PeterJCLaw** [comment]
> I will likely be doing a container trip either mid Feb or March to stock check (**not** to grab things in preparation). If it's on this issue, I'll check it.

@RealOrangeOne would you have capacity to fetch some small things from the container soonish? For example we need the various display Pis from the container at the earliest opportunity so that they can be configured ahead of time. Hopefully they're also small enough that they can be picked up without much faff (they're all in one box, it's literally just a case of grabbing that one RUB).

**Comment by RealOrangeOne** [comment]
Possibly, yes.

Sadly, It's likely now my trip won't happen until early/mid March. Alternatively, depending on timings, it might make sense to do this after the virtual competition.

**Comment by PeterJCLaw** [comment]
@RealOrangeOne do you think we can pin down a date for when you'll be able to do this? For the Pis at least this really needs to be pretty soon so we have time to validate the deployment of the screens. If you don't think you'll be able to do this before, say, the 18th I think we need to look at alternatives.

**Comment by RealOrangeOne** [comment]
- We have acquired the display Pis
- There are 15 of our usual flag fittings, plus a further 100. The flags made last year were also there.
- We have looked at how much tape we have
- There are clipboards in the container, and some pens. I did not test the pens, but there should be enough. 
- Hi-viz jackets are there
- There is 1 set of competition arena markers
- The normal table clothes and the fancy one are there
- The red lanyards are in a plastic binder
